config/templates/stack-template.json

```diff
@@ -17,19 +17,19 @@
           "{{$subnet.AvailabilityZone}}"
           {{end}}
         ],
-        "DesiredCapacity": "{{.WorkerCount}}",
+        "DesiredCapacity": "{{.WorkerASG.MinSize}}",
```

Comment:
The intention of this change is that the new `WorkerASG.MinSize` has the same value as the older `WorkerCount`. So if we want to omit it for that case, I would just omit it altogether and that would clarify that we currently do not support configuration of `DesiredCapacity`. What do you think?

It depends whether you want the `DesiredCapacity` to be set to the exact value it is prior to this change which it is `MinSize` is always set, defaulting to `WorkerCount`. In turn, this also keeps `stack-template.json` clean as it only uses the ASG properties rather than having multiple levels of defaulting/switching inside code and templates.